book/src/background/curves.md

```diff
@@ -19,25 +23,48 @@ handling: we find the line tangent to the point, and then find the single other
 intersects this line and then negate. Otherwise, in the event that a point is being
 "added" to its negation, the result is the point at infinity.
 
-The ability to add and double points naturally gives us a way to scale them by integers.
-The number of points on the curve is known as the "group order". If this number is prime
-$q$, we call the numbers that we scale curve points by "scalars" and consider that they
-are all elements of a scalar field $\mathbb{F}_q$.
+The ability to add and double points naturally gives us a way to scale them by integers,
+called _scalars_. The number of points on the curve is the group order. If this number
+is a prime $q$, then the scalars can be considered as elements of a _scalar field_,
+$\mathbb{F}_q$.
 
 Elliptic curves, when properly designed, have an important security property. Given two
 random elements $G, H \in E(\mathbb{F}_p)$ finding $a$ such that $[a] G = H$, otherwise
 known as the discrete log of $H$ with respect to $G$, is considered computationally
 infeasible with classical computers. This is called the elliptic curve discrete log
 assumption.
 
+If an elliptic curve group $\mathbb{G}$ has prime order $q$ (like the ones used in Halo),
```

Comment:
"used in Halo 2"? (since I assume you're referring to this crate)

Reply:
```suggestion
If an elliptic curve group $\mathbb{G}$ has prime order $q$ (like the ones used in Halo 2),
```